file.md

```diff
@@ -0,0 +1 @@
+New file
```

Comment:
Looks like this file is a little light on content. Care to expand a bit?

Reply:
Good idea. I fleshed this out a little.